packages/mini_sprite_editor/lib/config/cubit/config_cubit.dart

```diff
@@ -37,8 +47,7 @@ class ConfigCubit extends HydratedCubit<ConfigState> {
 
     return ConfigState(
       themeMode: themeMode,
-      filledColor: Color(json['filled_color'] as int),
-      unfilledColor: Color(json['unfilled_color'] as int),
+      colors: (json['colors'] as List).map((e) => Color(e as int)).toList(),
       backgroundColor: Color(json['background_color'] as int),
       mapGridSize: json['map_grid_size'] as int,
```

Comment:
Should u make some form of migration for this? Considering it is a hydrated bloc?

Reply:
Fair point! I did a handling for that and wrote a test for it.